Automattic/woocommerce-payments
# Re-removed deprecated depositsStatus key from account status

Fixes https://github.com/Automattic/woocommerce-payments/issues/4710

#### Changes proposed in this Pull Request

Removed deprecated deposit_status key from Account status.

Note:
- This change was from https://github.com/Automattic/woocommerce-payments/pull/8376.
- Was reverted by https://github.com/Automattic/woocommerce-payments/pull/8432 because broke a unit test.
- The unit test needs to be fixed. I filed an issue here https://github.com/Automattic/multiwoo/issues/1125.
- This PR should remained blocked until https://github.com/Automattic/multiwoo/issues/1125 is fixed.

#### Testing instructions

* Make sure this change does not break any deposit functionality.
* Make sure unit test mentioned in https://github.com/Automattic/multiwoo/issues/1125 passes.

-------------------


**Post merge**
